### PR TITLE
Add CONTRIBUTING.md document

### PR DESCRIPTION
**Pull request by carlosgalvezp**
In order to have a common view on the workflow.

### CODE REVIEW

**Review by orenmeiri — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Every developer should follow the following workflow when contributing to the
+project:
+
+Initial setup
+-------------
+
+To setup your environment, follow these steps:
+
+1. Go to the [`CarND-Eurobots/CarND-Capstone`](https://github.com/CarND-EuroBots/CarND-Capstone) repository and click on `Fork`. This will create a fork of the repo in
```
Comment (idanov) [inline CONTRIBUTING.md]: Is there an option to open pull requests without forking the project? Could we just create our feature branches in the main repository and open pull requests there?

**Review by SharkyZg — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (carlosgalvezp) [reply CONTRIBUTING.md]: Sure! The Fork option is what I've seen in many projects but I guess in our case it's fine to just create branches in the main repository. I also find it quite cumbersome to have a Fork especially when updating from upstream.

I'll change that!

Comment (carlosgalvezp) [reply CONTRIBUTING.md]: Done :)

Comment (idanov) [reply CONTRIBUTING.md]: Great!

**Pull request merged**